src/MainWindow.vala

```diff
@@ -3,61 +3,57 @@
  * SPDX-FileCopyrightText: 2021 elementary, Inc. (https://elementary.io)
  */
 
-public class Music.MainWindow : Hdy.ApplicationWindow {
+public class Music.MainWindow : Gtk.ApplicationWindow {
     construct {
-        Hdy.init ();
+        // try {
+        //     var css_provider = new Gtk.CssProvider ();
+        //     css_provider.load_from_data ("@define-color accent_color @ORANGE_500;");
 
-        try {
-            var css_provider = new Gtk.CssProvider ();
-            css_provider.load_from_data ("@define-color accent_color @ORANGE_500;");
+        //     Gtk.StyleContext.add_provider_for_display (Gdk.Display.get_default (), css_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+        // } catch (Error e) {
+        //     critical ("Can't set accent color: %s", e.message);
+        // }
 
-            Gtk.StyleContext.add_provider_for_screen (Gdk.Screen.get_default (), css_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
-        } catch (Error e) {
-            critical ("Can't set accent color: %s", e.message);
-        }
-
-        var queue_header = new Hdy.HeaderBar () {
+        var queue_header = new Adw.HeaderBar () {
+            css_classes = {"default-decoration", "flat"},
```

Comment:
css_classes is generally a bad idea - it replaces css classes isntead of adding them. It's not too bad _here_, but e.g. will break buttons - as buttons have `.image-button` and other classes.

```vala
queue_header.get_css_class ("default-decoration");
queue_header.get_css_class ("flat");
```